core/src/scene/dataLayer.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "scene/sceneLayer.h"
+#include <string>
+
+namespace Tangram {
+
+// DataLayer represents a top-level layer in the stylesheet, distinct from
+// SceneLayer by its association with a collection within a DataSource
+class DataLayer {
```

Comment:
Will it be better to make DataLayer a specialized class of SceneLayer by making it the child of SceneLayer?

Currently its implemented using "containership" OOPs concept. 
http://www.differencebetween.com/difference-between-inheritance-and-vs-containership/